src/utils/Draft.js

```diff
@@ -31,41 +31,40 @@ function arrangePicks(picks) {
   draft.picks[0] = [...picks];
 }
 
-const fetchLands = [
-  'Arid Mesa',
-  'Bloodstained Mire',
-  'Flooded Strand',
-  'Marsh Flats',
-  'Misty Rainforest',
-  'Polluted Delta',
-  'Scalding Tarn',
-  'Verdant Catacombs',
-  'Windswept Heath',
-  'Wooded Foothills',
-];
+const fetchLands = {
+  'Arid Mesa': ['W', 'R'],
+  'Bloodstained Mire': ['B', 'R'],
+  'Flooded Strand': ['W', 'U'],
+  'Marsh Flats': ['W', 'B'],
+  'Misty Rainforest': ['U', 'G'],
+  'Polluted Delta': ['U', 'B'],
+  'Scalding Tarn': ['U', 'R'],
+  'Verdant Catacombs': ['B', 'G'],
+  'Windswept Heath': ['W', 'G'],
+  'Wooded Foothills': ['R', 'G'],
```

Comment:
Should probably include Prismatic Vista here as colorless so it shows up as a fetch.

Reply:
Can you make a separate issue for this? Also include the ELD rainbow fetch